charts/cytomine/templates/postgresql/postgresql.yaml

```diff
@@ -30,34 +30,27 @@ spec:
       {{- end }}
       containers:
         - name: {{ .Chart.Name }}-postgis
-          image: "{{ .Values.postgresql.image.repository }}:{{ .Values.postgresql.image.tag }}"
-          imagePullPolicy: {{ .Values.postgresql.image.pullPolicy }}
+          image: "{{ .Values.images.postgresql }}"
+          imagePullPolicy: {{ .Values.images.pullPolicy }}
           volumeMounts:
             - name: postgres-storage
-              mountPath: "/postgres-data/"
-          command: ["/bin/bash"]
+              mountPath: "/var/lib/postgresql/data"
+          env:
+          - name: POSTGRES_USER
+            value: {{ .Values.postgresql.user }}
+          - name: POSTGRES_DB
+            value: {{ .Values.postgresql.database }}
+          - name: POSTGRES_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "cytomine.name" . }}-postgresql-secret
+                key: password
           # if the persistent volume seems empty, copy the data from the default
           # to the persistent volume, then start postgres using it.
-          args:
```

Comment:
I'm a bit unsure about this - it obviously works fine when testing locally. But for deploying in other environments I've had issues with the mount being a file system (with a `lost+found` popping up, causing postgres to become sad).